modules/cnf-about-topology-aware-lifecycle-operator-blocking-crs.adoc

```diff
@@ -0,0 +1,203 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-about-topology-aware-lifecycle-operator-blocking-crs_{context}"]
+= Blocking ClusterGroupUpgrade CRs
+
+You can create multiple `ClusterGroupUpgrade` CRs and control their order of application. 
+
+For example, if you create `ClusterGroupUpgradeC` that blocks the start of `ClusterGroupUpgradeA`, then `ClusterGroupUpgradeA` cannot start until the status of `ClusterGroupUpgradeC` becomes `UpgradeComplete`. 
+
+One `ClusterGroupUpgrade` CR can have multiple blocking CRs.
+
+.Prerequisites
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create {rh-rhacm} policies in the Hub cluster.
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CRs, and then save the YAML files in the `cgu-a.yaml`, `cgu-b.yaml`, and `cgu-c.yaml` files.
```

Comment:
```suggestion
. Create the `ClusterGroupUpgrade` CRs, and then save the YAML files in the `cgu-a.yaml`, `cgu-b.yaml`, and `cgu-c.yaml` files. Once all the CRs are enabled, their status should be as depicted below.
```

Reply:
@irinamihai 

> Once all the CRs are enabled, their status should be as depicted below.

I'm thinking that adding this^, the last step doesn't make sense. That's when `enable` is set to `true`. However, with the updated yaml samples, that is already done. 

I decided it would make more sense to keep the current yamls for step 1 and I'll add [these](https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/main/tests/kuttl/tests/blocking-mechanisms/01-assert.yaml) after enabling the CR as the `status.conditions.message` only gets updated after enabling the CR, I assume.